saleor/site/models.py

```diff
@@ -1,7 +1,13 @@
 from django.contrib.sites.models import _simple_domain_name_validator
 from django.db import models
-from django.utils.translation import pgettext_lazy
 from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import pgettext_lazy
+
+
+GOOGLE = 'google-oauth2'
+FACEBOOK = 'facebook'
+
+BACKENDS = [(FACEBOOK, 'Facebook-Oauth2'), (GOOGLE, 'Google-Oauth2')]
```

Comment:
I'd organize these constants and choices in the same way we have it other modules, for example see at the `OrderStatus` class in `saleor/order/__init__.py` for reference.